# Fix compilation with gpgme + lfs support
# Gentoo: https://bugs.gentoo.org/show_bug.cgi?id=275445
# Patch taken from LFS, thanks to Thierry Massenaux for pointing it out.
--- a/libseahorse/seahorse-gpg-options.c	2009-04-22 18:28:45 +0000
+++ b/libseahorse/seahorse-gpg-options.c	2009-07-24 11:52:49 +0000
@@ -19,6 +19,9 @@
  * Boston, MA 02111-1307, USA.
  */
 
+#include <config.h>
+#include "seahorse-util.h"
+
 #include <sys/param.h>
 #include <sys/wait.h>
 #include <sys/file.h>
@@ -32,7 +35,6 @@
 #include <sys/stat.h>
 
 #include "seahorse-gpgmex.h"
-#include "seahorse-util.h"
 #include "seahorse-gpg-options.h"
 
 #define  GPG_CONF_HEADER    "# FILE CREATED BY SEAHORSE\n\n"
--- a/libseahorse/seahorse-gpgmex-op.c	2009-04-22 18:28:45 +0000
+++ b/libseahorse/seahorse-gpgmex-op.c	2009-07-24 11:54:16 +0000
@@ -18,7 +18,9 @@
  * 59 Temple Place, Suite 330,
  * Boston, MA 02111-1307, USA.
  */
- 
+
+#include <config.h>
+
 #include <glib.h>
 #include <gpgme.h>
 #include <string.h>
--- a/agent/seahorse-agent-prompt.c	2009-04-22 18:28:45 +0000
+++ b/agent/seahorse-agent-prompt.c	2009-07-24 11:58:57 +0000
@@ -25,6 +25,8 @@
  * Public License.
  */
 
+#include <config.h>
+
 #include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
